docs/RunningFirstBenchmark.md

```diff
@@ -0,0 +1,114 @@
+# Running Your First Benchmark
+
+This tutorial is aimed at people new to the library and shows you how to run benchmarks with the Google Benchmark library.
+
+# Install the prerequisites
+
+We need the following programs:
+
+* git : to clone google benchmark and gtest libraries
+* cmake : to build google benchmark
+* make : to build google benchmark. Advanced users can use an alternative cmake backend like ninja.
+* g++ : to compile your c++ code. Advanced users can use other compilers like clang instead.
+
```

Comment:
cmake or bazel (bazel is preferred)
make or ninja or visual studio for windows
g++ or clang++ can be replaced by "A C++ compiler with C++11 support"